API/HelperClasses/EmailSender.cs

```diff
@@ -23,16 +23,14 @@ public interface IEmailSender
     /// </summary>
     public class EmailSender : IEmailSender
```

Comment:
Now that `API/HelperClasses/EmailSender.cs` has changed, maybe it is a good idea to add some small guide on how to use it correctly